app/Providers/Filament/AdminPanelProvider.php

```diff
@@ -51,17 +51,25 @@ public function panel(Panel $panel): Panel
                 'warning' => Color::Amber,
                 'blurple' => Color::hex('#5865F2'),
             ])
+            ->renderHook('panels::sidebar.footer', fn () => view('filament.Footer'))
+            ->navigationItems([
+                NavigationItem::make('client')
+                    ->label('Return to client')
```

Comment:
Can we name this `Exit Admin`
```suggestion
                    ->label('Exit Admin')
```